### ISSUE TITLE
doing inference using batch normalization with only one example

### ISSUE THREAD

**Issue by Prinsphield** [tensorflow/tensorflow#9796]
I build my model using batch normalization with `tf.layers.batch_normalization` . But when i am doing inference using saved model, I have to set the parameter `training=True`  in `tf.layers.batch_normalization` to make it work. Does it should be `training=False`?  Moreover, I have to feed a batch of test examples to make it work. It fails when fed with only one example. 
So what is the right way to use `tf.layers.batch_normalization`?
